### PR TITLE
🏗Add `#test` and `#testing` to alias list

### PR DESCRIPTION
**Pull request by rcebulko**
Don't look at the files tab, look at the first commit: https://github.com/ampproject/amphtml/pull/34853/commits/a9e25dcad1e0a4f550990fd9b3c10e72b4a4de6f
The rest of the changes are just a result of `amp lint --fix`

### CODE REVIEW

**Comment by rcebulko** [comment]
Don't look at the files tab, look at the first commit: a9e25dc
The rest of the changes are just a result of `amp lint --fix`

**Review by samouri — APPROVED** [review]

**Comment by rcebulko** [comment]
@kristoferbaxter bumping this in your queue. Only "real" diff is a9e25dc

File: build-system/babel-config/import-resolver.js [hunk 1]
```diff
@@ -18,11 +18,11 @@
 const fs = require('fs');
 const path = require('path');
 
-const JSCONFIG_PATH = path.join(__dirname, '..', '..', 'jsconfig.json');
+const JSCONFIG_PATH = path.join(__dirname, '..', '..', 'tsconfig.json');
```
Comment (kristoferbaxter) [inline build-system/babel-config/import-resolver.js:21]: Nit: Since this switched to tsconfig, might update the variable name here.

**Comment by rcebulko** [comment]
Real diff: 7ee3e067f8...5d1885c572

File: tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,10 @@
       "#polyfills/*": ["./src/polyfills/*"],
       "#preact/*": ["./src/preact/*"],
       "#purifier/*": ["./src/purifier/*"],
-      "#service/*": ["./src/service/*"]
+      "#service/*": ["./src/service/*"],
+
+      "#test/*": ["./test/*"],
+      "#testing/*": ["./testing/*"]
```
Comment (kristoferbaxter) [inline tsconfig.json:19]: Since this is a `tsconfig` now, do you need to mark `allowJS: true`?

**Review by kristoferbaxter — APPROVED** [review]
Looks good, one nit and one question to validate things work.

File: build-system/babel-config/import-resolver.js (hunk 1 above)
Comment (rcebulko) [reply build-system/babel-config/import-resolver.js]: Done

File: tsconfig.json (hunk 2 above)
Comment (rcebulko) [reply tsconfig.json:20]: Added, thank you (in the process of installing VSCode so I can test these changes myself)

**Comment by rcebulko** [comment]
Verified tsconfig changes locally

**Pull request merged**